### PR TITLE
Nuget package, first commit

### PR DESCRIPTION
**Pull request by vpenades**
As discussed in #134, In this pull request I dropped the basic idea of a nuget package.

Its basic usage would be this:

```c#
var report = glTF2Validator.ValidationContext.ValidateFile("somefile.gltf");

if (report.HasErrors) throw new Exception(report.RawReport);
```

In order to compile the project, it is required to compile the CLI tools beforehand, since they're embedded in the nuget package.

I have only tested the Windows path; theoretically it should work on Linux, but someone will need to do the testing.

I haven't defined anything for the semantic versioning. Ideally, the nuget package should have the same version ID as the gltf-validator itself.

### CODE REVIEW

**Comment by lexaknyazev** [comment]
Looks promising!

I'm concerned about relying API integration on text-based validation output. Its format slightly changes between releases and it does not include all the information that's available in a generated JSON report.

I could add a new command-line key that would print the JSON report to the `stdout` instead of writing it to a file (the latter is certainly not expected from API integration).

**Comment by vpenades** [comment]
> I could add a new command-line key that would print the JSON report to the `stdout` instead of writing it to a file (the latter is certainly not expected from API integration).

Certainly, having the json available to the stdout would greatly help.

**Comment by lexaknyazev** [comment]
CLI option has been added. Please rebase this PR onto `dev` branch and retarget it there.

**Comment by vpenades** [comment]
For some reason, it does not allow me to choose `dev `branch.

**Comment by lexaknyazev** [comment]
Just switched manually. Anyway, all files here are new, so it's safe.

File: nuget/ValidationResult.cs [hunk 1]
```diff
@@ -9,64 +11,49 @@ namespace glTF2Validator
...
-
-            var www = new List<string>();
-            var eee = new List<string>();
-
-            foreach (var l in lines)
+            var options = new JsonSerializerOptions
             {
-                if (l == "\tWarnings:") { status = 1; continue; }
-                if (l == "\tErrors:") { status = 2; continue; }
+                AllowTrailingCommas = true
```
Comment (lexaknyazev) [inline nuget/ValidationResult.cs]: Why is this needed? If the report contains trailing commas, it's a bug.

Comment (vpenades) [reply nuget/ValidationResult.cs]: Ah, I'll remove it if it's not needed.

**Comment by vpenades** [comment]
@lexaknyazev sorry, I've been working in a different machine, and I didn't notice the author email being used was very old. I've been trying to fix it but it's taking me too much time; so I'm going to remove this pull request and create a new one....

sorry for the inconvenience

**Comment by lexaknyazev** [comment]
No worries! 
FYI, it's possible to force-push a branch that is linked with a PR - GitHub handles that fine.
